README.md

```diff
@@ -60,20 +60,36 @@ copy the WORKSPACE snippet into your `WORKSPACE` file.
 
 ## Usage
 
-One-time re-format all files:
+⚡ One-time re-format all files:
 
 `bazel run @aspect_rules_format//format`
 
-Install as a git pre-commit hook:
+⚡ Re-format a specific file:
+
+`bazel run -- @aspect_rules_format//format some/file.ext`
+
+⚡ Install as a git pre-commit hook:
 
 ```bash
 $ echo "bazel run @aspect_rules_format//format" >> .git/hooks/pre-commit
 $ chmod u+x .git/hooks/pre-commit
 ```
 
-Check that files are already formatted, exit non-zero if formatting is needed:
+⚡ Install under pre-commit.com hook, in your `.pre-commit-config.yaml`:
+
+```yaml
+- repo: local
+  hooks:
+    - id: bazel-super-formatter
+      name: Format
+      language: system
+      entry: bazel run -- //:format fix
```

Comment:
Just to confirm. I always thought that the syntax was `bazel run //:format -- fix` where the double dash follows the target.

Reply:
you're right - if you have no "negative target patterns" like `-//other/...` then it doesn't matter, but your way is more idiomatic for it to follow the targets